src/vga.js

```diff
@@ -65,7 +65,7 @@ var VGA_HOST_MEMORY_SPACE_SIZE = Uint32Array.from([
  * @param {BusConnector} bus
  * @param {number} vga_memory_size
  */
-function VGAScreen(cpu, bus, vga_memory_size)
+function VGAScreen(cpu, bus, vga_memory_size, screen_options = undefined)
```

Comment:
I think the code would be a bit simpler if you force this to be an object. (VGAScreen is internal to v86, so we can change its interface arbitrarily).

Reply:
Patched it, I wasn't sure whether VGAScreen was considered internal.